cds/modules/deposit/static/templates/cds_deposit/types/video/form.html

```diff
@@ -89,7 +89,7 @@
           </div>
         </div>
           <!-- Published view -->
-          <div ng-show="$ctrl.cdsDepositCtrl.isPublished() && !showAll">
+          <div ng-show="$ctrl.cdsDepositCtrl.isPublished() || !showAll">
```

Comment:
`showAll` can be set to false only if `$ctrl.cdsDepositCtrl.isPublished()` is true, so isn't this the same as just `<div ng-show="$ctrl.cdsDepositCtrl.isPublished()">`? Shouldn't it be just `<div ng-show="!showAll">`?